Scripts/RunFunctionalTests.ps1

```diff
@@ -0,0 +1,71 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$PMCCommand,
+    [Parameter(Mandatory=$true)]
+    [int]$PMCLaunchWaitTimeInSecs,
+    [Parameter(Mandatory=$true)]
+    [int]$EachTestTimoutInSecs,
+    [Parameter(Mandatory=$true)]
+    [string]$NuGetDropPath,
+    [Parameter(Mandatory=$true)]
+    [string]$FuncTestRoot,
+    [Parameter(Mandatory=$true)]
+    [string]$RunCounter,
+    [Parameter(Mandatory=$true)]
+	[ValidateSet("15.0", "14.0", "12.0", "11.0", "10.0")]
+    [string]$VSVersion)
+
```

Comment:
Same as before. I want to enforce mandatory parameters for now so that it is strict and accurate.
Once the system is functioning properly on the CI, I will update the scripts to make it usable on a dev box as well. Would incorporate this comment then.